### PR TITLE
Feature/liwo 498

### PR DESCRIPTION
**Pull request by SiggyF**
Show information if multiple bands are returned

### CODE REVIEW

File: sig-deploy.sh [hunk 1]
```diff
@@ -41,9 +41,7 @@ pushd ${backend_dir}
 # download relevant sources
 svn export https://repos.deltares.nl/repos/LWB/trunk/sources/liwo.GeoServerTools
 svn export https://repos.deltares.nl/repos/LWB/trunk/sources/liwo.GeoServerToolsTest
```
Comment (CindyvdVries) [inline sig-deploy.sh:43]: Was also in other PR ?

File: src/lib/leaflet-utils/popup.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import L from '@/lib/leaflet-utils/leaf'
 import getFeatureInfo from '@/lib/get-feature-info'
 
-export function showLayerInfoPopup ({ map, activeLayer, unit, position, latlng }) {
+export function showLayerInfoPopup ({ map, activeLayer, unit, selectedLayer, position, latlng }) {
```
Comment (CindyvdVries) [inline src/lib/leaflet-utils/popup.js:4]: what is the difference between activeLayer and selectedLayer? comment?

File: src/lib/leaflet-utils/popup.js [hunk 3]
```diff
@@ -13,17 +13,38 @@ export function showLayerInfoPopup ({ map, activeLayer, unit, position, latlng }
     layer: activeLayer
   })
     .then(data => {
-      let value
+      let value = null
 
       if (data && data.properties && data.properties.GRAY_INDEX) {
```
Comment (CindyvdVries) [inline src/lib/leaflet-utils/popup.js:18]: if data.properties is not defined then this will give an error?

File: src/views/Combine.vue [hunk 4]
```diff
@@ -637,6 +637,7 @@ export default {
           map: mapObject,
           activeLayer: this.selectedVariantId,
```
Comment (CindyvdVries) [inline src/views/Combine.vue:638]: ah so activeLayer is acutally a ID?

Comment (SiggyF) [reply src/views/Combine.vue]: Fixed

**Pull request merged**